tox.ini

```diff
@@ -18,6 +18,7 @@ skip_install = true
 deps =
     setuptools < 52
     zc.buildout
+    zope.testrunner
```

Comment:
How did this work before?

Reply:
Probably because `zope.testrunner` is part of the `tests` extra.